keras/backend/mxnet_backend.py

```diff
@@ -953,8 +989,10 @@ def _normalize_axis(axis, ndim):
         for i, a in enumerate(axis):
             if a is not None and a < 0:
                 axis[i] = a % ndim
+    elif axis is None:
```

Comment:
Please check if my code match the following explanation?
---------
axis : None or int or tuple of ints, optional

Axis or axes along which a sum is performed. The default (axis = None) is perform a sum over all the dimensions of the input array. axis may be negative, in which case it counts from the last to the first axis.

New in version 1.7.0.

If this is a tuple of ints, a sum is performed on multiple axes, instead of a single axis or all the axes as before.